### PR TITLE
config: allow .yml as extension for config file

### PR DESCRIPTION
**Pull request by lukahartwig**
*Description*: Currently only config files ending with `.yaml` are treated as YAML files. If you use the common `.yml` extension it will try to parse it as JSON and result in a confusing error message for the user. Therefore this PR adds `.yml` as a valid file extension for config files.

*Risk Level*: low

*Testing*: manual test

*Docs Changes*: I added `.yml` in an example how to load config file via the CLI

*Release Notes*: N/A

### CODE REVIEW

File: source/common/json/json_loader.cc [hunk 1]
```diff
@@ -686,8 +686,8 @@ bool ObjectHandler::handleValueEvent(FieldSharedPtr ptr) {
 ObjectSharedPtr Factory::loadFromFile(const std::string& file_path) {
   try {
     const std::string contents = Filesystem::fileReadToEnd(file_path);
-    return absl::EndsWith(file_path, ".yaml") ? loadFromYamlString(contents)
-                                              : loadFromString(contents);
+    bool is_yaml = absl::EndsWith(file_path, ".yaml") || absl::EndsWith(file_path, ".yml");
```
Comment (mattklein123) [inline source/common/json/json_loader.cc]: nit: const bool

Comment (lukahartwig) [reply source/common/json/json_loader.cc]: done

**Comment by lukahartwig** [comment]
I did what you suggested and changed one of the examples so it uses `.yml` now

**Comment by mattklein123** [comment]
@lukahartwig please check CI and also needs a master merge.

**Comment by benmoss** [comment]
This would be nice, I just ran into this error when trying to use `.yml` as my file extension

**Pull request closed without merging**